model/src/test/kotlin/licenses/ResolvedLicenseTest.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model.licenses
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.collections.shouldHaveSize
+import io.kotest.matchers.shouldBe
+
+import org.ossreviewtoolkit.model.TextLocation
+
+class ResolvedLicenseTest : StringSpec({
+    "Copyright findings with the same year across different files should be maintained in the result" {
```

Comment:
I wonder if "with same year" may be misleading. 
Would it make sense to say:  "copyright findings which get processed to identical statements should be maintained"?

Reply:
Actually, the bug of unintended deduplication happened *before* the processing already. Which made me think that the test should also pass with `process = false`, which it didn't, due to another regression 😢 I've also fixed that one in a new commit; I'm really sorry for having introduced these. On the good side, I've now added more tests to guard against such things in the future.